internal/global/common/bleed.go

```diff
@@ -56,7 +56,7 @@ func init() {
 }
 
 func bleedPhase1(mod *modifier.Instance) {
-	state, ok := mod.State().(BleedState)
+	state, ok := mod.State().(*BleedState)
```

Comment:
to my pointer comment: only entanglement needed to change because was incorrect. Fine with others changing for consistency, but then you need to check the entire codebase for other places where we are creating the non break dots and make sure they are creating a dereferenced state otherwise that code will crash when ran